android/src/ti/map/PolylineProxy.java

```diff
@@ -169,6 +178,10 @@ public void processOptions()
 					processPatternDefinition(TiConvert.toInt(getProperty(PolylineProxy.PROPERTY_STROKE_PATTERN2))));
 			}
 		}
+
+		if (hasProperty(PolylineProxy.PROPERTY_JOINT_TYPE)) {
+			options.jointType(TiConvert.toInt(getProperty(PolylineProxy.PROPERTY_JOINT_TYPE)));
```

Comment:
Probably best to specify a default for TiConvert

Reply:
setting it to 0 (default)